examples/docker/assets/overrides.json

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) conda-store development team. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
This breaks JSON compatibility. Should we now rename this to `overrides.json5`?

Reply:
Could do